test/SourceKit/CursorInfo/rdar_36305791.swift

```diff
@@ -3,6 +3,7 @@ public class Observable<Element> {
 }
 Observable.create { }
 
+// Check that cursor info on create doesn't crash
```

Comment:
What is this actually returning and what changed in this PR?

Reply:
It's a request for actions, but there's none on the requested symbol. The test was added because it crashed as far as I can tell.

I can change it back now - I originally removed the kind when there were no symbols given back, but since added that back in due to client test failures.

Reply:
Turns out I can't change it back, I only add the kind when there *are* actions now (and this gives back none). I checked that this doesn't cause any issues so should be fine 👍